test/karma.conf.profile.js

```diff
@@ -0,0 +1,174 @@
+const path = require('path');
+const { chromium } = require('playwright');
+const webpack = require('webpack');
+
+const workspaceRoot = path.resolve(__dirname, '../');
+
+const browserStack = {
+  username: process.env.BROWSERSTACK_USERNAME,
+  accessKey: process.env.BROWSERSTACK_ACCESS_KEY,
+  build: `material-ui-${new Date().toISOString()}`,
```

Comment:
To allow seeing the difference with regular runs in BrowserStack?

```suggestion
  build: `material-ui-profile-${new Date().toISOString()}`,
```

Reply:
Nice idea. Working: https://automate.browserstack.com/dashboard/v2/builds/7c6d6dfd3c87df74b5d5083ad32d5c2be4f092f9